rbac/management/utils.py

```diff
@@ -279,6 +279,12 @@ def validate_group_name(name):
         raise serializers.ValidationError({key: _(message)})
 
 
+def check_duplicate_entry(grp_name, tenant_id):
```

Comment:
good job 🎉  it works and that's always important, I have a few minor comments
I would be very happy for a more descriptive function name here
+
another option to consider could be to catch the error from db to eliminate the not necessary call when provided group name is not duplicate

Reply:
Cool I will update with your recommended changes